ament_cmake_gmock/ament_cmake_gmock-extras.cmake

```diff
@@ -21,11 +21,17 @@ macro(_ament_cmake_gmock_find_gmock)
 
     find_package(ament_cmake_test QUIET REQUIRED)
 
+    set(_prefix "${ament_cmake_gmock_DIR}/../../..")
+
     if(NOT GMOCK_FOUND)
       # find gmock include and source folders
-      # fall back to system installed path (i.e. on Ubuntu)
-      set(_search_path_include "/usr/include/gmock")
-      set(_search_path_src "/usr/src/gmock/src")
+      # build the list of locations to search in reverse order
```

Comment:
Because the logic, as it was before, started by using `set(....)` and then doing front insertions. I just followed the same pattern, setting it to the least preferred option first and then adding other options optionally using front insertion.